### DISCUSSION TITLE
is there an equivalent of anacron

### DISCUSSION THREAD

**Discussion by dkebler** [jhuckaby/Cronicle#702] [Q&A] [answered]
Assuming say I have cronicle on a laptop.  Can cronicle run a job daily at a certain time but if the machine is off (e.g. a laptop) then run that job next time the machine is on?   This would be similar to anacron which I can use but was hoping to run cronicle for the benefit of the ui and for remote management.

**Comment by dkebler** [comment]
Maybe cronicle uses/can use systemd timers?  

they have an oncalendar and persistant settings which are very much like anacron.

https://linuxconfig.org/how-to-schedule-tasks-with-systemd-timers-in-linux

**Comment by jhuckaby** [comment, marked as answer]
Yes, this feature is built in.  You need to just need to enable "Run All" (Catch-Up) mode on your event.  Docs here:

https://github.com/jhuckaby/Cronicle/blob/master/docs/WebUI.md#run-all-mode
